tests/kernel/sql_tests.py

```diff
@@ -128,10 +128,6 @@ def test_arguments(self, mock_default_context):
     sql = google.datalab.bigquery.Query(query, values={}).sql
     self.assertEquals('SELECT * FROM [publicdata:samples.shakespeare]\n' +
                       'WHERE word IN ("thus", "forsooth")\nLIMIT 10', sql)
-    # As above but with overrides, using list
```

Comment:
There was one test that used kwargs, and one that used values.  Since we are removing the kwargs options, I removed the kwargs test.  The values test still covers the remaining behavior.